base_multi_company/models/company_activate.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class CompanyActivate(models.Model):
+
+    _name = 'company.activate'
+    _description = 'Company Activations'
+
+    is_active = fields.Boolean(
```

Comment:
@lasley I am wondering why you create a specific model while the property table is the perfect fit for that?